app/src/main/java/org/mozilla/fenix/onboarding/view/Onboarding.kt

```diff
@@ -206,7 +206,7 @@ private fun OnboardingSyncSignInContent() {
         Spacer(modifier = Modifier.height(32.dp))
 
         Text(
-            text = stringResource(id = R.string.onboarding_home_sync_title_2),
+            text = stringResource(id = R.string.onboarding_home_sync_title_3),
```

Comment:
```suggestion
            text = stringResource(R.string.onboarding_home_sync_title_3),
```

Reply:
Thought we were removing "id=" from everywhere

Reply:
I remember discussing with @MozillaNoah that we don't want to focus too heavily about removing the `id=` since by default the autocomplete in Android Studio will add this in for you, and it would probably be more work to just remove it manually. I would generally say we don't want to put too much energy in policing this and just accept that adding `id=` is okay.